### ISSUE TITLE
Step 2.a: Get concordium account nonce

### ISSUE THREAD

**Issue by jatitoam** [aesirxio/web3-id-preregistration-backend#3]
Similar to the demo, we need an `/account/:account/nonce` endpoint (GET) to generate a new random nonce, store it under an `accounts` collection and return it to the frontend with a json structure:

```json
{
   "nonce": "<nonce>"
}
```

If the account does not exist then it needs to be stored in the collection with its nonce.  If it exists, the nonce needs to be updated with a new random nonce.

Additionally, add a regex validation for concordium accounts.  If the account does not match the regex it should return a 406 with a json:

```json
{
   "error": "Account is not valid"
}
```

**Comment by thaoredweb** [comment]
@CongThuong Can you check again with my ip, it works when I create the first time but then it show Error: connect ECONNREFUSED 127.0.0.1:8083

**Comment by thaoredweb** [comment]
@CongThuong When I create a request GET with the account invalid
However, the response is 404 not found e, please check the message error two
![aer](https://user-images.githubusercontent.com/30283123/221148920-598a07b9-b3b7-474b-8c13-7194a55e5e91.png)

**Comment by thaoredweb** [comment]
{{domain}}/account/{{account}}/nonce
![step2a](https://user-images.githubusercontent.com/30283123/221491608-0c723d6b-38af-4627-add9-53ac7b1493de.png)

**Comment by thaoredweb** [comment]
When I use another address, it does not as expected:
With invalid address, it show the same message
![invalid](https://user-images.githubusercontent.com/30283123/221492455-c5804546-1331-414c-a76a-91735dd8b874.png)

**Comment by CongThuong** [comment]
it's a wrong API to get nonce chị @thaoredweb : {{domain}}/account/{{account}}/nonce . Please use the collection that I shared you

**Comment by thaoredweb** [comment]
@CongThuong There is no problem with the API e. only 1 thing the expected result does not correct

**Comment by CongThuong** [comment]
yeah, of course you cannot pass special characters directly to URL. it must be encoded first Chị @thaoredweb 
![image](https://user-images.githubusercontent.com/94885770/221773367-d9678a3b-666d-499d-922f-84f1c9738e2a.png)

**Comment by thaoredweb** [comment]
@jatitoam I have put an invalid address= 36FYT269JK4bqVVbwaXXzWBEZ9Q4DvR4GC6BiAwTk4JC7mUFGy#@
but the actual output does not work as noted "error": "Account is not valid"
Is that the issue he have to fix it?

**Comment by jatitoam** [comment]
Not a critical error... you can get a nonce but good luck after that validating the signature

**Comment by CongThuong** [comment]
We don't need signature to get nonce @jatitoam

**Comment by jatitoam** [comment]
No, of course.  But what I meant is simply that even if you can forge an account and get a nonce for it, you won't be able to get a valid signature of an invalid account, so it doesn't matter if the validation doesn't "work" here.
